update-requirements.py

```diff
@@ -0,0 +1,35 @@
+import subprocess
```

Comment:
Can we have a shebang here?

```suggestion
#!/usr/bin/env python

import subprocess
```

Reply:
I'd rather not since it messes up environment agnosticism

Reply:
Okay.